lametro/models.py

```diff
@@ -191,29 +191,58 @@ def upcoming_board_meeting(cls):
 
     @classmethod
     def current_meeting(cls):
-        # Create the boundaries for discovering events (in progess) within the timeframe stipulated 
-        # by Metro.
-        # Then, filter the events according to these boundaries.
-        six_minutes_from_now = datetime.now(app_timezone) + timedelta(minutes=6)
-        three_hours_ago = datetime.now(app_timezone) - timedelta(hours=3)
+        # QA Testing....
+        # fakenow = datetime(2018,1,18,10,10)
```

Comment:
Safe to strike this commented code?

Reply:
Actually should L194-L207 be here at all?

https://github.com/datamade/la-metro-councilmatic/blob/022b541525f26bcd1c685666f3fef966b71a11cf/lametro/models.py#L194-L207

I'd define `six_hours_ago` / `five_minutes` from now below the multi-line comment, and strike everything else.

Reply:
Yep - that's all junk.